bittide/src/Data/Constraint/Nat/Extra.hs

```diff
@@ -42,6 +42,13 @@ timesNDivRU' = unsafeCoerce (Dict :: Dict ())
 timesNDivRU'' :: forall a b . Dict (Div ((a * b) + (b - 1)) b ~ a)
 timesNDivRU'' = unsafeCoerce (Dict :: Dict ())
 
+clog2axiom :: CLog 2 (n * 2) :~: (CLog 2 n + 1)
+clog2axiom = unsafeCoerce Refl
```

Comment:
You know, I think we should append each of these proofs with a quickcheck property doctest: https://github.com/martijnbastiaan/doctest-parallel/#quickcheck-properties.

Reply:
Added in #170